### PR TITLE
Add device properties support to VirtualDeviceBuilder

### LINKED ISSUE #104: VirtualDeviceBuilder does not support setting device properties
As far as I can tell, evdev::uinput::VirtualDeviceBuilder doesn't provide any interface for advertising support for Properties.

See section device properties: <https://www.kernel.org/doc/Documentation/input/event-codes.txt>
My tablet's pen buttons currently are not functioning correctly potentially due to missing #101 and the inability to set properties altough this might be xournalpp's fault (as it is sending the button events) it would be easier to add support for more device configuration here.

I also succeeded in doing this with unsafe rust but I don't want that in other people's repos.

If wanted I can make a pr for this

### PR DESCRIPTION
**Pull request by ToxicMushroom**
fixes #104 
adds a with_properties method to VirtualDeviceBuilder in uinput.rs

### CODE REVIEW

File: src/uinput.rs [hunk 1]
```diff
@@ -7,10 +7,7 @@ use crate::constants::{EventType, UInputEventType};
 use crate::ff::FFEffectData;
 use crate::inputid::{BusType, InputId};
 use crate::raw_stream::vec_spare_capacity_mut;
-use crate::{
-    sys, AttributeSetRef, Error, FFEffectType, InputEvent, InputEventKind, Key, RelativeAxisType,
-    SwitchType, UinputAbsSetup,
-};
+use crate::{sys, AttributeSetRef, Error, FFEffectType, InputEvent, InputEventKind, Key, RelativeAxisType, SwitchType, UinputAbsSetup, PropType};
```
Comment (jeff-hiner) [inline src/uinput.rs:10]: This got longer, and the includes are out of alpha order. Did you run `cargo fmt` yet?

**Review by jeff-hiner — CHANGES_REQUESTED** [review]
Looks OK to me, but needs a `cargo fmt` pass.

File: src/uinput.rs (hunk 1 above)
Comment (ToxicMushroom) [reply src/uinput.rs]: thanks for noticing, I had intellij configured to run optimize imports on commit I think.

**Review by jeff-hiner — APPROVED** [review]

**Comment by jeff-hiner** [comment]
Looks like there are conflicts with your other PR. As soon as those are resolved, we should be good to merge this.

Thanks for your help!

**Comment by ToxicMushroom** [comment]
should be good to go now :) thanks for reminding me to fix the merge conflict

**Pull request merged**